new-lamassu-admin/src/pages/maintenance/MachineDetailsCard.js

```diff
@@ -0,0 +1,148 @@
+import { ReactComponent as DownloadReversedIcon } from '../../styling/icons/button/download/white.svg'
+
+import { Status } from '../../components/Status'
+import ActionButton from '../../components/buttons/ActionButton'
+
+const Label = ({ children }) => {
+  const useStyles = makeStyles(labelStyles)
+  const classes = useStyles()
+
+  return <div className={classes.label}>{children}</div>
+}
+
+const MachineDetailsRow = ({ machine, ...props }) => {
+  const useStyles = makeStyles(detailsRowStyles)
+  const classes = useStyles()
+
+  return (
+    <>
+      <div className={classes.wrapper}>
+
+        <div className={classnames(classes.row)}>
+          <div className={classnames(classes.col)}>
+            <div className={classnames(classes.row)}>
+              <div className={classnames(classes.col, classes.col2)}>
+                <div className={classes.innerRow}>
+                  <div>
+                    <Label>Statuses</Label>
+                    <div>
+                      {machine.statuses.map((status, index) => <Status status={status} key={index} />)}
+                    </div>
+                  </div>
+                  <div>
+                    {/* <div className={classes.commissionWrapper}> */}
+                    <Label>Lamassu Support article</Label>
+                    <div>
+                      {machine.statuses.map((status, index) => <span key={index} />)}
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div className={classnames(classes.col, classes.col2)} style={{ background: '#ebefff', width: '2px' }} />
+          <div className={classnames(classes.col)}>
+            <div className={classnames(classes.row)}>
+              <div className={classnames(classes.col, classes.col2)}>
+                <div className={classes.innerRow}>
+                  <div>
+                    <Label>Machine Model</Label>
+                    <div>{machine.machineModel}</div>
+                  </div>
+                  <div className={classes.commissionWrapper}>
+                    <Label>Address</Label>
+                    <div>{machine.machineLocation}</div>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div className={classnames(classes.row)}>
+              <div className={classnames(classes.col, classes.col2)}>
+                <div className={classes.innerRow}>
+                  <div>
+                    <Label>Paired at</Label>
+                    <div>{moment(machine.pairedAt).format('YYYY-MM-DD HH:mm:ss')}</div>
+                  </div>
+                  <div className={classes.commissionWrapper}>
+                    <Label>Software update</Label>
+                    <div className={classes.innerRow}>
+                      {machine.softwareVersion}
+                      &nbsp;
```

Comment:
We should prevent using this espaced sequence, we can do the same thing with css.